pkg/kubelet/container/helpers.go

```diff
@@ -310,7 +311,7 @@ func SandboxToContainerState(state runtimeapi.PodSandboxState) State {
 // GetContainerSpec gets the container spec by containerName.
 func GetContainerSpec(pod *v1.Pod, containerName string) *v1.Container {
 	var containerSpec *v1.Container
-	podutil.VisitContainers(&pod.Spec, podutil.AllFeatureEnabledContainers(), func(c *v1.Container, containerType podutil.ContainerType) bool {
```

Comment:
why this change? previously this explicitly excluded any container types whose corresponding feature was not enabled